CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 - #1021, Reduce join size in allColumns for faster program start - @nextstopsun
 - #411, Remove the need for pk in &select for parent embed - @steve-chavez
 - #1016, Fix anonymous requests when configured with jwt-aud - @ruslantalpa
+- #925, Fix RPC high memory usage by using parametrized query - @steve-chavez
```

Comment:
Anything to add about the handling of overloaded functions as well?

Reply:
I mentioned earlier [here](https://github.com/begriffs/postgrest/pull/1005#discussion_r155926177) that there wasn't a behavior change related to overloaded functions.

However thinking it again, now the overloaded functions depend on the `dbStructure` to be correctly detected, guess I could add that under a `Changed` header in the CHANGELOG.